libs/platform/ebpf_epoch.h

```diff
@@ -74,7 +74,8 @@ extern "C"
 
     /**
      * @brief Allocate an epoch work item; a work item that can be scheduled to
-     * run when the current epoch ends.
+     * run when the current epoch ends. Allocated work items must either be
+     * scheduled or cancelled.
```

Comment:
```suggestion
     * scheduled or canceled.
```
American vs British spelling :)  American is one L